### PR TITLE
Improve error handing in misc.js and steal.js

### PR DESCRIPTION
**Pull request by perryprog**

### CODE REVIEW

File: src/commands/currencyCommands/steal.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
 const { GenericCommand } = require('../../models/')
 let min = 500
 
+let dmStolenUser = async (Memer, user, msg, worth) => {
```
Comment (Devoxin) [inline src/commands/currencyCommands/steal.js]: Why not const?

File: src/utils/misc.js [hunk 2]
```diff
@@ -1,45 +1,44 @@
 const config = require('../config.json')
 
-module.exports = {
-  errorMessages: async (e) => {
-    let errors = {}
+let errors = {
```
Comment (Devoxin) [inline src/utils/misc.js]: No const?

File: src/utils/misc.js [hunk 3]
```diff
@@ -63,13 +62,13 @@ module.exports = {
 
   parseTime: (time) => {
     const methods = [
-      { name: 'd', count: 86400 },
-      { name: 'h', count: 3600 },
-      { name: 'm', count: 60 },
-      { name: 's', count: 1 }
+      {name: 'd', count: 86400},
```
Comment (Devoxin) [inline src/utils/misc.js]: Spaces please, looks a little cleaner + it's consistent with the rest of the code's style

File: src/utils/misc.js [hunk 4]
```diff
@@ -63,13 +62,13 @@ module.exports = {
...
+      {name: 'h', count: 3600},
+      {name: 'm', count: 60},
+      {name: 's', count: 1}
     ]
 
-    const timeStr = [ Math.floor(time / methods[0].count).toString() + methods[0].name ]
+    const timeStr = [Math.floor(time / methods[0].count).toString() + methods[0].name]
```
Comment (Devoxin) [inline src/utils/misc.js]: Spaces here also

File: src/utils/misc.js (hunk 3 above)
Comment (perryprog) [reply src/utils/misc.js]: oops, auto format must have touched that

**Pull request merged**